docs/ug/authorConfigSyntax.md

```diff
@@ -0,0 +1,80 @@
+{% set title = "Appendix: `author-config.csv` advanced syntax" %}
+<frontmatter>
+  title: "{{ title | safe }}"
+</frontmatter>
+
+{% from 'scripts/macros.njk' import embed, step with context %}
+
+<h1 class="display-4"><md>{{ title }}</md></h1>
+
+<div class="lead">
+
+Given below are the advanced syntax available for `author-config.csv`. 
+</div>
+
+## Multiple `Repository's Location` per author
+
+#### Specifying multiple `Repository's Location`
+
+To specify multiple `Repository's Location` in a single row, we can use a semicolon `;` as a separator.
+
+Below is an example:
+
+| Repository's Location                                                               | Branch  | Author's Git Host ID | ... Hidden columns    |
+|-------------------------------------------------------------------------------------|---------|----------------------|-----------------------|
+| https://github.com/reposense/RepoSense.git;https://github.com/MarkBind/markbind.git | master  | sikai00              | --                    |
+
+We have now furnished author details for both repositories in a single row, instead of having two individual rows per `Repository's Location`.
+
+<br>
+
+#### Specifying `Repository's Location` with one or more specific branches
+
+To list out one or more branches for a particular `Repository's Location`, we can use a pipe `|` separator to demarcate the different branches.
+
+Below is an example:
+
+| Repository's Location                                                | Branch   | Author's Git Host ID | ... Hidden columns    |
+|----------------------------------------------------------------------|----------|----------------------|-----------------------|
+| https://github.com/reposense/RepoSense.git~master\|release\|cypress  | master   | sikai00              | --                    |
```

Comment:
You may want to mention in the UG which column takes precedence if branch names are given in both the location and branch columns.